README.md

```diff
@@ -150,7 +1444,12 @@ vars:
   hubspot_service_enabled: true                           # Enables all service/ticket models. Default = false
   hubspot_ticket_deal_enabled: true                       # Default = false
 ```
+
+### Dbt-core Version Requirement for disabling freshness tests
```

Comment:
We need to include the proper reference to dbt when mentioning it within our docs.
```suggestion
### dbt Core™ Version Requirement for disabling freshness tests
```

Reply:
removed it wholly